src/board_controller/ant_neuro/ant_neuro.cpp

```diff
@@ -96,7 +96,8 @@ int AntNeuroBoard::prepare_session ()
         safe_logger (spdlog::level::info, "eego sdk version is: {}.{}.{}.{}",
             fact->getVersion ().major, fact->getVersion ().minor, fact->getVersion ().micro,
             fact->getVersion ().build);
-        amp = fact->getAmplifier ();
+        amp = fact->getAmplifier (board_descr["default"]["name"], params.serial_number);
```

Comment:
there is `getAmplifiers` in the ant neuro sdk, I think it would be better to use that instead and iterate over the list and check for serial number and name if needed. This way will also lead to better loging of erros like wrong name, wrong serial number, etc.